packages/flutter/lib/src/material/tabs.dart

```diff
@@ -712,7 +712,6 @@ class _TabBarState<T> extends ScrollableState<TabBar<T>> implements TabBarSelect
     setState(() {
       _tabBarSize = tabBarSize;
       _tabWidths = tabWidths;
-      _updateScrollBehavior();
```

Comment:
Why was this line wrong? It seems like if they change, the scroll behavior also needs to change, no?

Also, now that there's only one client for _updateScrollBehavior, and since it's only one statement long, let's just inline it where it's called.